frontend/src/libs/components/link/link.tsx

```diff
@@ -1,15 +1,29 @@
 import { NavLink } from 'react-router-dom';
 
 import { type AppRoute } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
 import { type ValueOf } from '#libs/types/types.js';
 
+import styles from './styles.module.scss';
+
 type Properties = {
   to: ValueOf<typeof AppRoute>;
   children: React.ReactNode;
+  className?: string | undefined;
```

Comment:
```suggestion
  className?: string;
```
'?' already means it can be 'undefined'

Reply:
I did a test where I passed classes to the Link component and it showed me a lint error that is removed when undefined is explicitly defined in the type.

Here is the error: 

The type '{ children: element; to:"/"; className: string | undefined; }' cannot be assigned to the 'Properties' type with 'exactOptionalPropertyTypes: true'. Consider adding 'undefined' to the target property types.

The way to fix it would be every time you import a style do it like this: 

<Link to={AppRoute.ROOT} className={styles['image'] ?? ''}>

But I don't know, I think it looks a bit weird. I don't know if it's better to declare undefined explicitly.

Reply:
className?: string; is exactly the same as className?: string | undefined;
? means ```type | undefined```
So there's no need to mention 'undefined' once again there

Reply:
yes, but when you test this ability, the error I mentioned appears